tests/data/power_flow/source-big-sk/input.json

```diff
@@ -39,15 +39,17 @@
         "status": 0,
         "u_ref": 1.0,
         "sk": 2e50,
-        "rx_ratio": 0.1
+        "rx_ratio": 0.1,
+        "z01_ratio": 2.0
```

Comment:
To double check, since you're changing values in the test in parallel with updating the code: is this correct? Default `z01_ratio` is `1.0`

Reply:
This was done on purpose. This way, as pointed out by @nitbharambe, the output will be the same, but now the off-diagonal (`m`) elements of the admittance matrix will be and non-zero. The intention was to make sure that no other instability errors would propagate, and if so, that the cut-off can mitigate it.

Reply:
maybe good to make 2 separate test cases of this

Reply:
@mgovers Is it needed though? In the end the results are the same, this just makes the admittance matrix not a perfect diagonal one.

Reply:
Perfect and non-perfect are different edge cases (location where rounding error occurs)

Reply:
This has been left untouched, i.e. `z01_ratio = 1` as it is not really relevant to the instability issue.